libs/distancing.py

```diff
@@ -32,8 +33,12 @@ def __init__(self, config, source, live_feed_enabled=True):
         self.face_mask_classifier = None
 
         self.running_video = False
-        self.tracker = CentroidTracker(
-            max_disappeared=int(self.config.get_section_dict("PostProcessor")["MaxTrackFrame"]))
+        self.tracker = IOUTracker(
+            max_lost=int(self.config.get_section_dict("PostProcessor")["MaxLost"]),
+            iou_threshold=self.config.get_section_dict("PostProcessor")["TrackerIOUThreshold"],
```

Comment:
```suggestion
            iou_threshold=float(self.config.get_section_dict("PostProcessor")["TrackerIOUThreshold"]),
```
If we don't cast the `TrackerIOUThreshold` as float, the processor will through the exception:
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/repo/libs/engine_threading.py", line 46, in run
    self.engine.process_video(self.source['url'])
  File "/repo/libs/distancing.py", line 317, in process_video
    cv_image, objects, distancings = self.__process(cv_image)
  File "/repo/libs/distancing.py", line 125, in __process
    tracks = self.tracker.update(detection_bboxes, class_ids, detection_scores)
  File "/repo/libs/trackers/iou_tracker.py", line 68, in update
    if iou(self.tracks[track_id].bbox, bb) > self.iou_threshold:
TypeError: '>' not supported between instances of 'float' and 'str'
```